docs/modules/widgets/pages/widgets.adoc

```diff
@@ -0,0 +1,89 @@
+= Widget: Compose Material
+
+This section exposes the custom UI Widget we've already developed so you don't have to reinvent the wheel.
```

Comment:
"exposes custom UI composable widgets" (no "the", and widgets plural and without capital)